.github/workflows/tokens.yml

```diff
@@ -0,0 +1,67 @@
+name: Generate CSS variables
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
```

Comment:
Do we need to run this on a schedule instead of manually?

Reply:
I remove it.